common/src/models/magic_wand/magic_wand.cc

```diff
@@ -55,16 +55,21 @@ MagicWandResult magic_wand_classify() {
   };
 }
 
+static int interpret_score(uint32_t& score) {
+  return 1000000 * static_cast<int>(*reinterpret_cast<float*>(&score));
```

Comment:
This will always return 1000000 or 0 because the score is in the range [0.0, 1.0]. If we move the position of the multiplication we can get a more precise print-out.
```suggestion
  return static_cast<int>(*reinterpret_cast<float*>(&score) * 1000000);
```